alchemy-web/src/content/docs/providers/cloudflare/worker.mdx

```diff
@@ -115,516 +210,376 @@ And register `env.d.ts` in your `tsconfig.json`'s `types`.
+:::
+
+**Consumer implementation:**
 ```ts
-import { Worker } from "alchemy/cloudflare";
+// ./src/processor.ts
+import type { MessageBatch } from "@cloudflare/workers-types";
 
-const worker = await Worker("nodejs-worker", {
-  name: "nodejs-worker",
-  entrypoint: "./src/api.ts",
-  compatibility: "node", // Automatically includes nodejs_compat flag
-});
+export default {
+  async queue(batch: MessageBatch, env: Env) {
+    for (const message of batch.messages) {
+      try {
+        const data = message.body;
+        await processTask(data);
+        message.ack(); // Acknowledge successful processing
+      } catch (error) {
+        message.retry(); // Retry on failure - respects maxRetries
+      }
+    }
+  }
+};
 ```
 
-### Available Presets
+**When to use:** Background job processing, webhook handling, email processing, image optimization, data synchronization.
 
-- **`"node"`**: Enables Node.js compatibility by including the `nodejs_compat` flag
+## Smart Placement (performance optimization)
 
-### Combining with Custom Flags
-
-Preset flags are automatically combined with any custom `compatibilityFlags` you provide:
+Enable automatic network optimization by configuring smart placement to reduce latency and improve performance:
 
 ```ts
-import { Worker } from "alchemy/cloudflare";
-
-const worker = await Worker("advanced-worker", {
-  name: "advanced-worker", 
+export const optimizedWorker = await Worker("api", {
   entrypoint: "./src/api.ts",
-  compatibility: "node", // Adds nodejs_compat
-  compatibilityFlags: ["experimental_feature"], // Additional flags
-  // Result: ["nodejs_compat", "experimental_feature"]
+  placement: {
+    mode: "smart"  // Automatically optimize placement for performance
+  }
 });
 ```
 
-## Smart Placement
-
-Enable smart placement to automatically optimize your worker's placement across Cloudflare's global network:
-
-```ts
-import { Worker } from "alchemy/cloudflare";
+**Benefits:**
+- ✅ Automatic network optimization based on performance metrics
+- ✅ Reduced latency for global users
+- ✅ No infrastructure management required
+- ✅ Seamless scaling across Cloudflare's network
 
-const worker = await Worker("optimized-worker", {
-  name: "optimized-worker",
-  entrypoint: "./src/api.ts",
-  placement: {
-    mode: "smart", // Enable smart placement
-  },
-});
-```
+**When to use:** Global applications, latency-sensitive APIs, high-traffic workloads, applications with geographically distributed users.
 
 :::note
-Learn more about [Smart Placement](https://developers.cloudflare.com/workers/configuration/smart-placement/) in the Cloudflare documentation.
+You can learn more about [Smart Placement](https://developers.cloudflare.com/workers/configuration/smart-placement/) in the Cloudflare documentation.
 :::
 
-## CPU Time Limit
+## Local development & configuration
```

Comment:
Should this be higher up? seems more important than smart placement

Reply:
Certainly can move it. No particular reason not too